src/components/__tests__/SideNav.test.tsx

```diff
@@ -4,7 +4,7 @@ import SideNav from "../SideNav";
 
 describe("SideNav", () => {
     const wrapper = shallow(<SideNav />);
-    it("renders a button with \"Toggle\" as children", () => {
+    it('renders a button with "Toggle" as children', () => {
```

Comment:
This change caused by ' npm run ci' command, and it will cause a single quote warning by eslint.
So please change it to "renders a button with 'Toggle' as children"